### ISSUE TITLE
Nose is skipping tests when both @generate and @attr decorators have been used.

### ISSUE THREAD

**Issue by plomakin** [kevinastone/generator#1]
Test code example:

```
  @attr(some_attr=['foo', 'bar'])
  @generate('name', 'description', 'enabled')
   def test_migrate_something(self, param):
...
```

Nose command example:
`nosetests -v -d --attr=some_attr=foo ./`
Nose output:

```
...
----------------------------------------------------------------------
XML: /home/username/usercompany/project/nosetests.xml
----------------------------------------------------------------------
Ran 0 tests in 0.001s

OK
```

Using ubuntu 14.04 and python package versions:

```
$ virtualenv --version
15.0.1
$ pip --version
pip 8.1.1 from /home/username/usercompany/project/.venv/local/lib/python2.7/site-packages (python 2.7)
$ pip freeze
test-generator==0.1.1
nose==1.3.7
nose-testconfig==0.9.1
```

**Comment by plomakin** [comment]
I believe this might help somehow:
http://stackoverflow.com/questions/7727678/nose-ignores-test-with-custom-decorator

**Comment by kevinastone** [comment]
the issue is that `@attr` scribbles on the test method.  I can fix this to copy it over to the generated test methods.  This is pretty ugly though.  It should have scribbled something like `test_method.__nose_attribs__` instead of writing attributes directly on the method instance.

**Comment by plomakin** [comment]
If you don't wanna merge this, just put a pull request, I will reuse it in my methods

**Comment by kevinastone** [comment]
You could try #2 to see if that works.

**Comment by plomakin** [comment]
#2 is resolving issue, if you don't want to merge it to master branch, then just keep nose-attribs branch. I will simply specify then it in my requirements.txt like this: 
`git+https://github.com/kevinastone/generator.git@nose-attribs#egg=test-generator`

**Comment by kevinastone** [comment]
I'd use the commit rather than the branch:

```
git+https://github.com/kevinastone/generator.git@dc9c3a5ed5e8ba70f8d514e847b882f11807bd57#egg=test-generator
```

**Comment by jaspreetrawel** [comment]
Below command installs code from this pull request successfully. I used pip 8.1.2.

"pip install git+https://github.com/kevinastone/generator.git@dc9c3a5ed5e8ba70f8d514e847b882f11807bd57#egg=test-generator"

But when I package test-generator as part of my module using setup.py, it always fetches test-generator from pypi. One solution was to specify the above url in dependency links but that also did not work because dependency links processing is deprecated in pip 8.1.2.

Any ideas from your side on any other known solutions?

Also can you change the package version in the pull request to 0.1.2?

**Comment by kevinastone** [comment]
Released version `0.1.2` with the nose attrib fixes from #2.

**Comment by jaspreetrawel** [comment]
Thanks @kevinastone
